### PR TITLE
Fix alembic migration history

### PR DESCRIPTION
**Pull request by MoritzWeber0**
# Description

Depends on #267 and extends #209. It fixes some migration issues.

# Testing
With our new test from #267.

### CODE REVIEW

**Comment by MoritzWeber0** [comment]
Is continued in #285.

File: backend/capellacollab/__main__.py [hunk 1]
```diff
@@ -31,7 +31,6 @@ def filter(self, record: logging.LogRecord) -> bool:
 
 logging.getLogger("uvicorn.access").addFilter(HealthcheckFilter())
 log.addFilter(HealthcheckFilter())
-logging.getLogger(__name__).setLevel(config["logging"]["level"])
```
Comment (MoritzWeber0) [inline backend/capellacollab/__main__.py:34]: This line is still needed if the Collaboration Manager is started from a context that is not alembic.

File: backend/capellacollab/alembic/versions/d888f5c30aab_add_model.py [hunk 2]
```diff
@@ -101,7 +104,7 @@ def upgrade():
             slug = base_slug
             index = 0
 
-            while slug in existing_slugs:
+            while slug in existing_slugs[project_id]:
```
Comment (amolenaar) [inline backend/capellacollab/alembic/versions/d888f5c30aab_add_model.py:107]: What is this doing? A comment is desired.

**Review by amolenaar — APPROVED** [review]
One bit of code is unclear. Apart from that it's a nice improvement.

File: backend/capellacollab/alembic/versions/d888f5c30aab_add_model.py (hunk 2 above)
Comment (MoritzWeber0) [reply backend/capellacollab/alembic/versions/d888f5c30aab_add_model.py:107]: It is the migration of the old databases. Per project, the slug of the model has to be unique. This was not a constraint in the old version, therefore there are conflicts (and we need to assign new slugs, adding just `-\d` to the existing "slugs").

File: backend/capellacollab/alembic/env.py [hunk 3]
```diff
@@ -15,24 +17,28 @@
...
     config.set_main_option("sqlalchemy.url", cfg["database"]["url"])
 
-# Interpret the config file for Python logging.
-# This line sets up loggers basically.
-if config.attributes.get("configure_logger", True):
-    fileConfig(config.config_file_name)
-
 # Import models
-import capellacollab.sql_models  # isort:skip
-from capellacollab.sql_models import extensions  # isort:skip
```
Comment (TimoBbz) [inline backend/capellacollab/alembic/env.py:30]: Why not importing the models anymore there?

**Review by TimoBbz — APPROVED** [review]

**Pull request merged**